documentation/modules/managing/con-broker-config-properties.adoc

```diff
@@ -200,20 +201,39 @@ log.roll.ms=604800000
-Segments also become eligible for deletion less often.
+These settings can be overridden at the topic level using `segment.bytes` and `segment.ms`. 
+The choice to lower or raise these values depends on the policy for segment deletion. 
+A larger size means the active segment contains more messages and is rolled less often. 
+Segments also become eligible for deletion less frequently.
 
-You can set time-based or size-based log retention and cleanup policies so that logs are kept manageable.
-Depending on your requirements, you can use log retention configuration to delete old segments.
-If log retention policies are used, non-active log segments are removed when retention limits are reached.
-Deleting old segments bounds the storage space required for the log so you do not exceed disk capacity.
+In Kafka, log cleanup policies determine how older log data is managed.
+In most cases, you won't need to change the default configuration at the cluster level, which specifies the `delete` cleanup policy and enables the log cleaner used by the `compact` cleanup policy:
 
-For time-based log retention, you set a retention period based on hours, minutes and milliseconds.
-The retention period is based on the time messages were appended to the segment.
+[source,env]
+----
+# ...
+log.cleanup.policy=delete
+log.cleaner.enable=true
+# ...
+----
 
-The milliseconds configuration has priority over minutes, which has priority over hours. The minutes and milliseconds configuration is null by default, but the three options provide a substantial level of control over the data you wish to retain. Preference should be given to the milliseconds configuration, as it is the only one of the three properties that is dynamically updateable.
+Delete cleanup policy:: Delete policy is the default cluster-wide policy for all topics.
+The policy is applied to topics that do not have a specific topic-level policy configured. 
+Kafka removes older segments based on time-based or size-based log retention limits. 
+Compact cleanup policy:: Compact policy is generally configured as a topic-level policy (`cleanup.policy=compact`).
+Kafka's log cleaner applies compaction on specific topics, retaining only the most recent value for a key in the topic.
+You can also configure topics to use both policies (`cleanup.policy=compact,delete`). 
+
+.Setting up retention limits for the delete policy
+
+The delete policy corresponds to managing logs with data retention policies. 
+The policy is suitable when data does not need to be retained forever.
+You can establish time-based or size-based log retention and cleanup policies to keep logs manageable. 
+
+When log retention policies are employed, non-active log segments are removed when retention limits are reached.
+This deletion of old segments helps prevent exceeding disk capacity.
+
+For time-based log retention, you set a retention period based on hours, minutes, and milliseconds: 
```

Comment:
I think it should be "or milliseconds" as if you set `log.retention.hours`, `log.retention.minutes` and `log.retention.ms` to different values, the value of `log.retention.ms` takes priority